.travis.yml

```diff
@@ -7,7 +7,7 @@ matrix:
   include:
     - script: ./build.sh verify-swiftlint
       env: JOB=verify-swiftlint
-      before_install: brew update; brew upgrade swiftlint
+      before_install: brew update; brew upgrade swiftlint || true
```

Comment:
We could instead only run `brew upgrade swiftlint` if it's outdated:

    brew update; brew outdated swiftlint || brew upgrade swiftlint